mobile/lib/presentation/widgets/people/person_tile.widget.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:immich_mobile/extensions/build_context_extensions.dart';
+import 'package:immich_mobile/pages/common/large_leading_tile.dart';
+import 'package:immich_mobile/services/api.service.dart';
+import 'package:immich_mobile/utils/image_url_builder.dart';
+
+// TODO: Only pass person object, instead of id and name when PersonDto and DriftPerson are unified
+class PersonTile extends StatelessWidget {
```

Comment:
To help with the review: I've extracted this widget from `people_picker.dart` to be reused there and in the merge alert dialog